PolicyProject1/src/main/java/com/example/demo/model/request/GetPolicyRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.example.demo.model.request;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
```

Comment:
This code will enforce Jackson to serialize json properties in SnakeCaseStrategy.
SnakeCaseStrategy – properties will be in snake_case format. Eg: first_name